ex04.ex

```diff
@@ -55,7 +55,7 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(xs), do: reduce(xs,&(if &1<&2,do: &1, else: &2))
```

Comment:
# -2

I would like to see code written without `if`.

~~~ elixir
def min(xs), do: reduce(xs, &min_of/2)
def min_of(a, b) when a < b, do: a
def min_of(a, b) do: b
~~~

Now, I know that `when` is no different to `if`. But doing it this way means that each function `min` and `min_of` does just one thing.